ui/src/literature/components/__tests__/LiteratureItem.test.jsx

```diff
@@ -55,4 +55,81 @@ describe('LiteratureItem', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('renders bulletpoint if conference_info and publication_info present', () => {
```

Comment:
you don't care about bullet points anymore, having one test case where you have all props with full data (all fields of literature that are used) and one case with only required ones are enough I think